plugins/circleci/package.json

```diff
@@ -40,8 +40,9 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "4.0.0-alpha.45",
     "circleci-api": "^4.0.0",
-    "dayjs": "^1.9.4",
+    "humanize-duration": "^3.27.0",
```

Comment:
Unfortunately Luxon doesn't provide a way to humanize durations.

Reply:
I wonder if that might warrant an appendix to the ADR so there is sort of a best practice, that if other's want to use Luxon they know a good pair to it to keep the deps consistent would be `humanize-duration` also?  just a thought; maybe the core maintainers have a thought on that, but yeah this makes sense.

Reply:
That sounds like a good idea :+1: I'll make a note on the RFC issue about the ADR. 

I did come across [an issue on Luxon](https://github.com/moment/luxon/issues/688) around humanising durations and intervals, it's currently open and linking to a proposal for adding standards to the Intl API for durations.